README.md

```diff
@@ -55,6 +55,10 @@ Start up two clients:
 
     ./rose-client mydriver.py
 
+The server FQDN/IP address can be specified that way:
+
+    ./rose-client --server-address example.com mydriver.py
+
```

Comment:
The previous example show starting a server at <server-address>, so the client example should use the same server address.

Lets amend the previous example instead of adding a new one.

Kids are not expected to know what his FQDN or IP address, best just use the term "server address".  To make this clear, we should show an example like:

    ./rose-client --server-address 1.2.3.4 my_driver.py

Since most server will not have a FQDN, best show an IP address in the example.

Reply:
Done